ts/utils/input.ts

```diff
@@ -128,3 +133,15 @@ export function getCreditCardFromState(
 
   return some(card);
 }
+
+/**
+ * This function checks if the cardholder charset is ammitted.
+ * We consider not a valid character an accented character.
+ *
+ * @param cardHolder
+ */
+export const isValidCardHolder = (cardHolder: Option<string>) =>
+  cardHolder.fold(false, cH => {
+    const cardHolderWithoutDiacriticalMarks = _.deburr(cH);
+    return _.isEqual(cH, cardHolderWithoutDiacriticalMarks);
```

Comment:
since they are two strings, strict equality should be enough. it isn't so?

Reply:
Yes, solved in [7976f68](https://github.com/pagopa/io-app/pull/2833/commits/7976f68ab91c6bbf4328548763e2534f9dd88f33)